polkadot/node/core/pvf/README.md

```diff
@@ -0,0 +1,45 @@
+# PVF Host
+
+This is the PVF host, responsible for responding to requests from Candidate
+Validation and spawning worker tasks to fulfill those requests.
+
+See also:
+
+- for more information: [the Implementer's Guide][impl-guide]
+- for an explanation of terminology: [the Glossary][glossary]
+
+## Running basic tests
+
+Running `cargo test` in the `pvf/` directory will run unit and integration
+tests.
+
+**Note:** some tests run only under Linux and/or with the `ci-only-tests`
+feature enabled.
+
+See the general [Testing][testing] instructions for more information on
+**running tests** and **observing logs**.
+
+## Running a test-network with zombienet
+
+Since this crate is consensus-critical, for major changes it is highly
+recommended to run a test-network. See the "Behavior tests" section of the
+[Testing][testing] docs for full instructions.
+
+To run the PVF-specific zombienet test:
+
+```sh
+RUST_LOG=parachain::pvf=trace zombienet --provider=native spawn zombienet_tests/functional/0001-parachains-pvf.toml
+```
+
+## Testing on Linux
+
+Much of the PVF functionality, especially related to security, is Linux-only. If
+you touch anything security-related, make sure to test on Linux! If you're on a
+Mac, you can either run a VM or you can hire a VPS and use [EternalTerminal][et]
+to ssh into it. (ET preserves your session across disconnects, and unlike mosh
```

Comment:
```suggestion
to ssh into it. (ET preserves your session across disconnects, and unlike most shells
```

Reply:
didn't know mosh was a tool, I thought it's a typo 😆

Reply:
Haha! I have this note here because et is still unknown compared to mosh, but either of these tools are *huge* improvements over ssh.